news/2024-05-15-numpy-2-migration.md

```diff
@@ -0,0 +1,9 @@
+# NumPy 2 migration
+
+In preparation for the NumPy 2 release, we have begun a migration of all
+conda-forge packages that depend on `numpy` during the build (so in
+`requirements/host`). The migration status is tracked
+[on our status page](https://conda-forge.org/status/migration/numpy2).
+Particular package's status can be found there. Specific recommendations for
+maintainers updating packages can be found in the migration PRs themselves.
+Please reach out through the usual channels if you have any questions.
```

Comment:
I agree. I don't think most maintainers know what a repodata patch is / the purpose of one. Linking to the repodata-patches feedstock main page isn't directly helpful either. I think linking directly to the README in the recipe folder is better because it explains how to make a patch, but the README is missing context of what a patch is and why it is useful.

Reply:
Yeah it would be good to include all this sort of information somewhere

Personally would rather keep this brief and then just link resources

Honestly find an issue with more detail sufficient for that. Hence suggesting it above ( https://github.com/conda-forge/conda-forge.github.io/pull/2185#discussion_r1602636666 ). Though we could link several issues here with short descriptions based on needs

Reply:
Agree with keeping the announcement short and referring to somewhere where we collect more details.

Reply:
Ok. Feel free to copy the text wherever you want to put it.

Reply:
Does someone want to have a go at opening a webpage issue with this info?